build_all.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# this script replaces the all.yml playbook due to a variable scoping issue
+# that appeared with Ansible 2.0.0.2
+SCRIPT_DIR=$(cd $(dirname $0) && pwd)
+
+function PlayBookRunner {
+    playbook=$1;
+    config=$2;
+    provisioner=$3;
+    sudo ansible-playbook -i hosts $playbook -e "config_file=$config provisioner=$provisioner"
+}
+
+# initrd
+arg1=common/initrd_wrapper.yml;
```

Comment:
nit: arg# does not tell me much (and I can see when I look at the PlayBookRunner line). "playbook" instead of arg1, "config" instead of arg2, and "provisioner" instead of arg3?